editor/test/integration/model_test.clj

```diff
@@ -49,13 +50,13 @@
       (testing "can assign single dae file as animations"
         (let [dae-resource (workspace/resolve-workspace-resource workspace "/mesh/treasure_chest.dae")]
           (test-util/with-prop [node-id :animations dae-resource]
-            (is (= #{"treasure_chest"} (set (map :id (:animations (g/node-value node-id :animation-set)))))))))
+            (is (= #{(murmur/hash64 "treasure_chest")} (set (map :id (:animations (g/node-value node-id :animation-set)))))))))
```

Comment:
Should these be hashes, though? We won't be able to present these in the editor, or report name conflicts or mention an animation by name in any way.

How does it work to select the default animation from a dropdown if these are hashes?